### PR TITLE
Create fee-structure.md

### PR DESCRIPTION
**Pull request by oliverbeige**

### CODE REVIEW

**Comment by fleupold** [comment]
I think this is really well written. What kind of feedback are you looking for exactly?

I think the intro is good, and am really looking forward to how this applies concretely for the dƒusion.

File: fee-structure/fee-structure.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+3. ADAPTIVE A functional form is fitted to the (history of) observed states: τ = f(x, t,...). 
+4. GENERATIVE A generative algorithm anticipates resource needs and allocates resources based on the (history of) observed states: τ = τ0 iff f(x, t), τ1 else.
+
+### Governance mechanism toolbox
+
+Any complex governance mechanism is a combination of multiple governance modes and implementations. For instance, auctions as competitive governance modes come in multiple forms: Dutch, English, first-price, second-price, etc. Considerable effort has been put into understanding the advantages and drawbacks of each mode, but intermodal comparisons are still in their infancy.
+
+For instance, while market vs hierarchy is a well-established intermodal comparison, it does not take into account the “enterprization” of markets which created a major impetus for the emergence of blockchain. The ability to set the rules and control the information flow creates rent extraction opportunities under non-zero exit costs.
+
+-	VERTICAL: dictatorship, hierarchy, bureaucracy, walking away, …
```
Comment (bh2smith) [inline fee-structure/fee-structure.md]:
```suggestion
-	**Vertical**: dictatorship, hierarchy, bureaucracy, walking away, …
```

File: fee-structure/fee-structure.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+-	COMPETITIVE: market, auction, bargaining, effort, …
```
Comment (bh2smith) [inline fee-structure/fee-structure.md]:
```suggestion
-	**Competitive**: market, auction, bargaining, effort, …
```

File: fee-structure/fee-structure.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+-	PARTICIPATORY: voting, consensus, ...
```
Comment (bh2smith) [inline fee-structure/fee-structure.md]:
```suggestion
-	**Participatory**: voting, consensus, ...
```

File: fee-structure/fee-structure.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+-	RANDOMIZED: lottery, discovery, ...
```
Comment (bh2smith) [inline fee-structure/fee-structure.md]:
```suggestion
-	**Randomized**: lottery, discovery, ...
```

File: fee-structure/fee-structure.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+To design a governance mechanism then requires making tradeoffs: use simple but inflexible parameters where simplicity is required, and complex parametrizations where flexibility is required.
+
+As a design principle, fixing parameters to a constant value is a preferable first (“minimum viable”) step as long as realistic testing is possible. Testing should hypothesize and verify failure modes of constant value parametrization before implementing more complex mechanisms.
+
+A sequence of parametrization steps by adding complexity could look as follows. 
+
+1. FIXED A single, invariant value x is assigned a priori to a parameter: τ = x.
```
Comment (bh2smith) [inline fee-structure/fee-structure.md]:
```suggestion
1. **Fixed**: A single, invariant value x is assigned a priori to a parameter: τ = x.
```

File: fee-structure/fee-structure.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+2. VARIABLE The parameter is expressed as a function of state parameters: τ = f(x). The choice of the function is predefined a priori.
```
Comment (bh2smith) [inline fee-structure/fee-structure.md]:
```suggestion
2. **Variable**: The parameter is expressed as a function of state parameters: τ = f(x). The choice of the function is predefined a priori.
```

File: fee-structure/fee-structure.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+3. ADAPTIVE A functional form is fitted to the (history of) observed states: τ = f(x, t,...).
```
Comment (bh2smith) [inline fee-structure/fee-structure.md]:
```suggestion
3. **Adaptive** A functional form is fitted to the (history of) observed states: τ = f(x, t,...). 
```

File: fee-structure/fee-structure.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+To design a governance mechanism then requires making tradeoffs: use simple but inflexible parameters where simplicity is required, and complex parametrizations where flexibility is required.
+
+As a design principle, fixing parameters to a constant value is a preferable first (“minimum viable”) step as long as realistic testing is possible. Testing should hypothesize and verify failure modes of constant value parametrization before implementing more complex mechanisms.
+
+A sequence of parametrization steps by adding complexity could look as follows. 
+
+1. FIXED A single, invariant value x is assigned a priori to a parameter: τ = x.
+2. VARIABLE The parameter is expressed as a function of state parameters: τ = f(x). The choice of the function is predefined a priori.
+3. ADAPTIVE A functional form is fitted to the (history of) observed states: τ = f(x, t,...). 
+4. GENERATIVE A generative algorithm anticipates resource needs and allocates resources based on the (history of) observed states: τ = τ0 iff f(x, t), τ1 else.
```
Comment (bh2smith) [inline fee-structure/fee-structure.md]:
```suggestion
4. **Generative** A generative algorithm anticipates resource needs and allocates resources based on the (history of) observed states: τ = τ0 iff f(x, t), τ1 else.
```

**Comment by bh2smith** [comment]
Congrats @oliverbeige on your first git merge!

**Pull request merged**